inst/include/Rcpp/generated/Function__operator.h

```diff
@@ -26,108 +26,108 @@
 
 	template <TYPENAMES>
 	SEXP operator()(ARGUMENTS){
-		return Rcpp_fast_eval(Rf_lang2(Storage::get__(), pairlist(PARAMETERS)), R_GlobalEnv);
+        return invoke(pairlist(PARAMETERS), R_GlobalEnv);
 	}
 
 */
 	template <typename T1>
 	SEXP operator()(const T1& t1) const {
-		return Rcpp_fast_eval(Rcpp_lcons(Storage::get__(), pairlist(t1)), R_GlobalEnv);
```

Comment:
the result of `pairlist()` was not protected, and `Rcpp_lcons()` allocates, so it might gc() it. 
the result of `Rcpp_lcons()` is not protected, so it might be claimed when `Rcpp_fast_eval()` allocates ... 

all changes in that file are the same.